### DISCUSSION TITLE
Any German speaking dev available for a podcast?

### DISCUSSION THREAD

**Discussion by denbec** [supabase/supabase#3336] [Questions] [answered]
Hi guys, I'm Dennis from [programmier.bar](https://www.programmier.bar), a German podcast for web and app devs with ~5.000 listeners. Some of our listeners have asked us to do an episode about supabase. We're very excited to talk to someone who has already gotten good experience in using it. And unfortunately, it has to be in German. Do _you_ want to join us or have a good suggestion?

Don't know if it's alright to post it here 🤷‍♂️ Thanks!

**Comment by thorwebdev** [comment, marked as answer]
Moin @denbec, zum Glück gibt's auch immer mal einen Deutschen im Team ;) Schickst du mir mehr Details per Mail an thor[at]supabase.io?
